tests/unit/parser.js

```diff
@@ -239,28 +317,128 @@ tap.test('Parser', {autoend: true}, t => {
         t.end()
       }
     )
+
+    t.test(
+      'should parse body from json to xml format and create an orchestration - with orchestrations array',
+      t => {
+        const ctx = {
+          orchestrations: [], // empty array to skip condition check
+          body: {
+            Name: 'Jet',
+            Surname: 'Li',
+            Email: 'jet@openhim.org'
+          },
+          state: {
+            uuid: 'randomUidForRequest',
+            metaData: {
+              name: 'Testing endpoint'
+            }
+          },
+          set: (key, value) => {
+            ctx.header = {}
+            ctx.header[key] = value
+          },
+          request: {
+            header: {
+              'x-openhim-transactionid': '12333'
+            }
+          }
+        }
+        const outputFormat = 'XML'
+        const expectedHeader = {
+          'Content-Type': 'application/xml'
+        }
+
+        const expectedBody =
+          '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>\n<root>\n' +
+          '  <Name>Jet</Name>\n' +
+          '  <Surname>Li</Surname>\n' +
+          '  <Email>jet@openhim.org</Email>\n</root>'
```

Comment:
@BMartinos Minor: Storing these long string in an array and joining it up when needed could be a more readable way of storing XML in some cases. It means we don't need the newline characters included in the strings. Thoughts?

```suggestion
        const expectedBody = [
          '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>',
          '<root>',
          '  <Name>Jet</Name>',
          '  <Surname>Li</Surname>',
          '  <Email>jet@openhim.org</Email>',
          '</root>'
        ].join('\n')
```